static/templates/404.html

```diff
@@ -9,5 +9,8 @@ <h1>Not Found</h1>
     <p>
       The requested URL "{{url}}" was not found on this server.
     </p>
+    <p>
+      Do you want to search for <a href="{{root}}/search?content={{bookName}}&pattern={{url_slug}}">{{url_slug}}</a> in this zim file?
+    </p>
```

Comment:
This must now be passed as `{{details}}`. Also, rephrase the sentence like [requested by @kelson42](https://github.com/kiwix/kiwix-tools/issues/430#issuecomment-802651937)